/**:
  ros__parameters:
    smooth_velocity_before_planning: true  # [-] if true, smooth the velocity profile of the input trajectory before planning

    trajectory_polygon_collision_check:
      decimate_trajectory_step_length : 2.0 # longitudinal step length to calculate trajectory polygon for collision checking
      goal_extended_trajectory_length: 6.0

      # consider the current ego pose (it is not the nearest pose on the reference trajectory)
      # Both the lateral error and the yaw error are assumed to decrease to zero by the time duration "time_to_convergence"
      # The both errors decrease with constant rates against the time.
      consider_current_pose:
        enable_to_consider_current_pose: true
        time_to_convergence: 1.5 #[s]
